enhancements/operator-sdk/downstream-operator-sdk.md

```diff
@@ -0,0 +1,281 @@
+
+# make local commit
+git commit -m "Merge upstream tag $version" -m "Merge executed via ./UPSTREAM-MERGE.sh $version $upstream_remote $rebase_branch" -m "$(printf "Overwritten conflicts:\\n%s" "$unmerged_files")"
+
+# verify merge is correct
+git --no-pager log --oneline "$(git merge-base origin/"$rebase_branch" tags/"$version")"..tags/"$version"
+
+printf "\\n** Upstream merge complete! **\\n"
+echo "View the above incoming commits to verify all is well"
+echo "(mirrors the commit listing the PR will show)"
+echo ""
+echo "Now make a pull request, after it's LGTMed make the tag:"
+echo "$ git checkout $rebase_branch
+$ git pull
+$ git tag <origin version>-$version
+$ git push origin <origin version>-$version"
+```
+
+#### Handling Patches
+
+Typically we want to submit all bugs upstream, have them release then sync the release downstream. But there will be times we need to fix a problem downstream to resolve a bug that we can’t wait for an upstream release. The process here will be twofold.
+
+1. For fixes to vendored dependencies, have a patch in the patches directory that will be applied to the vendor after we run go mod vendor.
+1. For fixes to mainline code, fix the mainline code directly. NOTE: these changes will be lost when re-running UPSTREAM-MERGE.sh script.
+
+For either of the above, we will always submit the patches to the appropriate upstream repos.
+
+#### Directory Structures
+
+The following is a list of directories and files that we will add to the repos:
+
+* UPSTREAM-MERGE.sh # script used to sync the upstream to the downstream
+* ci
+  * dockerfiles
+    * variety of Dockerfiles used to build images for running tests
+* tests
+  * e2e test scripts
+* prow.Makefile # Makefile used by CI
+* patches
+  * contain the patch files that may be carried
+* release
+  * ansible
+    * ansible_collections # contains the sync ansible collections
+    * bin # entrypoint and other binaries for ansible base images
+    * Dockerfiles # various dockerfiles
+  * helm
+    * bin # entrypoint and other binaries for helm base images
+    * Dockerfiles # various dockerfiles
+  * sdk
+    * bin # entrypoint and other binaries for sdk base images
+    * Dockerfiles # various dockerfiles
+* vendor
+  * contains the dependencies for the project
+
+
+### Risks and Mitigations
+
+This is a low risk enhancement since we already have existing processes for handling downstream builds.
+
+## Design Details
+
+### Open Questions [optional]
+
+### Test Plan
+
+The existing set of Operator SDK tests will be run downstream. Today we run tests for the Ansible and Helm operators.
+
+### Graduation Criteria
+
+Operator SDK will go out once the downstream builds work.
+
+### Upgrade / Downgrade Strategy
+
+All Operator SDK releases have a migration guide on what users have to do to upgrade their operators. The downstream repos will not have upgrades or downgrades.
+
+### Version Skew Strategy
```

Comment:
Will we be using the release branches provided by the DPTP/ART automation?

Reply:
> Will we be using the release branches provided by the DPTP/ART automation?

Yes we will. I've updated the enhancement to reflect this.